vslib/inc/TrafficForwarder.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "swss/sal.h"
+
+#include <stddef.h>
+#include <sys/socket.h>
+
+namespace saivs
+{
+    class TrafficForwarder
+    {
+    public:
+        virtual ~TrafficForwarder() = default;
+
+    protected:
+        TrafficForwarder() = default;
+
+        void add_vlan_tag(
```

Comment:
seems to me likt those methods can be static, also use camelcase method naming

Reply:
Done.